src/stylesheet-registry.js

```diff
@@ -94,8 +94,8 @@ export default class StyleSheetRegistry {
     this.remove(props)
   }
 
-  flush() {
-    this._sheet.flush()
+  flush(options = {}) {
```

Comment:
what is this for?

Reply:
In case someone is using this method and needs to set the nonce.

Reply:
I don't think it is necessary also `StyleSheet#flush` is not using those options https://github.com/zeit/styled-jsx/blob/368a6e4e3e1211e84b6bc4bb4db44dcd426a0bd9/src/lib/stylesheet.js#L192

Can you revert this change?